internal/core/cobra_utils_test.go

```diff
@@ -0,0 +1,48 @@
+package core
+
+import (
+	"context"
+	"reflect"
+	"testing"
+)
+
+type testType struct {
+	Name string
+}
+
+func testGetCommands() *Commands {
+	return NewCommands(
+		&Command{
+			Namespace: "test",
+			ArgSpecs: ArgSpecs{
+				{
+					Name: "name-id",
+				},
+			},
+			ArgsType: reflect.TypeOf(testType{}),
+			Run: func(ctx context.Context, argsI interface{}) (i interface{}, e error) {
+				return "", nil
+			},
+		},
+	)
+}
+
+func Test_handleUnmarshalErrors(t *testing.T) {
+	t.Run("underscore", Test(&TestConfig{
+		Commands: testGetCommands(),
+		Cmd:      "scw test name_id",
+		Check: TestCheckCombine(
```

Comment:
I don't think golden are appropriate for this kind of test.
This should be somtehing like
```
Check: TestCheckError(&CliError{
   ...
}
```